.golangci.yml

```diff
@@ -5,7 +5,9 @@ issues:
   new: true
 
 linters:
-  enable-all: true
+  enable:
+    - gofmt
+    - golint
```

Comment:
Why not run all?

Reply:
Upon further reading, I see the `enable-all` option being deprecated soon:
https://golangci-lint.run/usage/configuration/

Can we set the following to start with?
```
enable:
    - goconst
    - gocyclo
    - gofmt
    - goimports
    - golint
    - govet
    - misspell
    - unused
    - whitespace
```

Reply:
Yes happy to do that. Some of these are enabled by default but it doesn't hurt to explicitly have our list laid out here.